pkgs/tools/filesystems/darling-dmg/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, lib, cmake, fuse, zlib, bzip2, openssl, libxml2, icu } :
+
+stdenv.mkDerivation rec {
+  name = "darling-dmg";
+  src = fetchFromGitHub {
+    owner = "darlinghq";
+    repo = "darling-dmg";
+    rev = "9522e3907b82f6cde141b3e0e0063f09c8710cbf";
+    sha256 = "5d968b0609f9bfbecc26753d21b5182e15183b672967f7dec6038404cd6a6d7f";
+  };
+  buildInputs = [ cmake fuse openssl zlib bzip2 libxml2 icu ];
+  cmakeConfigureFlagFlags = ["-DCMAKE_BUILD_TYPE=RELEASE"];
+  meta = {
+    homepage = http://www.darlinghq.org/;
+    description =
+    lib.strings.concatStringsSep " " ["Darling lets you open OS X dmgs on Linux"];
+    platforms = stdenv.lib.platforms.linux;
+    license = stdenv.lib.licenses.gpl3;
+  };
```

Comment:
Wanna add yourself as a maintainer? That'll let hydra notify you if it breaks and so on.